test/build.sh

```diff
@@ -139,7 +139,7 @@ declare -a executables_rrfs_utl_created=( adjust_soiltq.exe \
     EXEC_DIR=${BIN_DIR}/bin
     if [ $build_it -eq 0 ] ; then
       ./devbuild.sh --platform=${machine} --compiler=${compiler} --build-dir=${BUILD_DIR} --install-dir=${BIN_DIR} \
-        --clean --rrfs || fail "Build ${machine} ${compiler} FAILED"
+        --clean all || fail "Build ${machine} ${compiler} FAILED"
```

Comment:
Does this `--clean` behave the same as it did before? Should we have both `--clean` and `--remove`?

Reply:
Thanks for catching that bug! `--clean` does a `make clean` on the target you selected. I am not sure if `--remove` that does `rm -rf build` is needed now but I kept it just in case.